src/EditorFeatures/Core.Wpf/InlineParamNameHints/InlineParamNameHintsTagger.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
```

Comment:
copyright.  (i'll stop saying this from thsi point on.  but in general, apply previous feedback to later files as well).